package.json

```diff
@@ -30,7 +30,7 @@
     "test:contracts:upgrade:2to3": "npm run start:blockchain:client:2to3 & truffle test ./test-upgrade/2to3/2to3.js",
     "test:contracts:gasCosts": "npm run start:blockchain:client & truffle migrate --reset --compile-all && truffle test test-gas-costs/gasCosts.js --network development",
     "test:contracts:patricia": "npm run start:blockchain:client parity & truffle migrate --reset --compile-all && truffle test packages/reputation-miner/patricia-test.js --network development",
-    "test:contracts:coverage": "SOLIDITY_COVERAGE=1 solidity-coverage && istanbul check-coverage --statements 99 --branches 94 --functions 99 --lines 99",
+    "test:contracts:coverage": "SOLIDITY_COVERAGE=1 truffle run coverage --network coverage --temp build && istanbul check-coverage --statements 99 --branches 94 --functions 99 --lines 99",
```

Comment:
There something important here: `--temp build` means: "write the temporary coverage artifacts to `build`". It treats that folder as disposable and will delete it after coverage runs. Normally we would use a folder called `.coverage_artifacts` but there are many places in the test setup scripts where the artifacts are requested from `build`. 

In <= `0.6.x` the whole project gets copied into a sub-folder. That works well here but has been the source of quite a bit of weirdness :) In the plugin version, the contracts/artifacts are written to temp folders at the root and truffle's path variables are re-targeted to use them as sources for compilation and testing.